README.rst

```diff
@@ -53,8 +53,15 @@ API Key at the bottom.
 * `Documentation at ReadTheDocs <https://ns1-python.readthedocs.org/en/latest/>`_
 * `NS1 REST API Documentation <https://ns1.com/api/>`_
 
+Tests
+=====
+
+Unit tests use `pytest` (`pip install pytest`). 2.7 also requires `mock` to be
```

Comment:
Awesome to get documentation around this.  We have a `tox.ini` in the repo to automate the venv setup and multi-env testing, maybe we should call that out here?

Reply:
good idea, done